homeassistant/helpers/deprecation.py

```diff
@@ -171,6 +171,8 @@ def _print_deprecation_warning_internal(
     description: str,
     verb: str,
     breaks_in_ha_version: str | None,
+    *,
+    log_when_no_integration_is_found: bool = True,
```

Comment:
I think we can make this false by default?

Reply:
Param is required now